### PR TITLE
UNMS-Restrction-Added

### PR DESCRIPTION
**Pull request by fgampel**
Plugin is not compatible with UNMS until now. Next months a new update will be realeased

To release a new version of a plugin, we suggest the following checklist. Checking these locally before pushing to the repository gives you immediate feedback, instead of waiting for the repository to automatically run the checks.

- [ ] update the information.version in manifest.json; please use [semantic versioning](https://semver.org/#faq) ("major.minor.patch", e.g. "1.4.2" - major can break compatibility, minor can introduce compatible new features, patch is for bug fixes)
- [ ] in the project root directory, run `bash php-cs-check.sh` to check for valid PHP code
  - normally, this should report no errors
  - if any found, please fix - these could prevent your plugin from working
- [ ] in the `plugins/` directory, run `php ../pack-plugin.php $YOUR_PLUGIN`, where $YOUR_PLUGIN is the directory name of your plugin; this creates the installable package
  - if you have a `composer.json`, this will check it and install the dependencies - if this reports as "outdated", please fix by running `composer update` in your plugin's directory
  - normally, this should finish without errors or warnings
- [ ] in the project root directory, run `php validate.php`
  - normally, this should report 0 errors
  - if it complains that `plugins.json` is outdated, run `php generate-json.php > plugins.json` to update it.
- [ ] commit the changes made during these steps

### CODE REVIEW

**Comment by fgampel** [comment]
Plugins.json was updated. I really don't understand what are Travis requirements.

**Comment by janprochazkacz** [comment]
Thank you for your work. The problem is only in line separators and encoding of plugins.json file. Currently is in `UTF-16LE BOM CRLF` and must be in `UTF-8 LF`.

File: plugins/argentina-afip-invoices/src/manifest.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
         "displayName": "Facturas Argentina AFIP",
         "description": "Plugin para obtener CAE (Facturacion electronica) en Argentina",
         "url": "https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/argentina-afip-invoices",
-        "version": "1.0.1",
+        "version": "1.0.2",
         "ucrmVersionCompliancy": {
             "min": "2.15.0-beta3",
             "max": null
         },
       "unmsVersionCompliancy": {
```
Comment (keksa) [inline plugins/argentina-afip-invoices/src/manifest.json]: Hello @fgampel, thank you for the PR. To make the plugin incompatible with UNMS, you can just remove the `unmsVersionCompliancy` from manifest altogether. Take a look here as an example https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/plugins/routeros-packet-manager/src/manifest.json#L9

**Comment by keksa** [comment]
@fgampel Feel free to update just the `manifest.json` file, we'll update the plugin archive and `plugins.json` after it's merged.

File: plugins/argentina-afip-invoices/src/manifest.json (hunk 1 above)
Comment (fgampel) [reply plugins/argentina-afip-invoices/src/manifest.json]: Thanks @keksa , in the past i always let this file without UNMS compatibility, but i don't know why, someone added this compatibility, and after that, i had many many emails in my inbox telling me that the plugin is not working haha!

Updated again, now i think plugin.json is in right codding.

Thanks

**Pull request merged**